src/components/ReportTransaction.js

```diff
@@ -42,10 +55,19 @@ class ReportTransaction extends Component {
             chatReportID: this.props.chatReportID,
             transactionID: this.props.action.originalMessage.IOUTransactionID,
             comment: '',
+            rejectedIDs: this.props.iouTransactionsBeingRejected,
         });
     }
 
+    isRejected(transactionID, rejectedIDs) {
+        if (!rejectedIDs) {
```

Comment:
I believe `rejectedIDs` will always be an array, so `!rejectedIDs` will always be false. Maybe you meant to check if the array is empty here?

Reply:
Opps! yeah, wanted to check if array is empty. replaced it with `!rejectedIDs || !rejectedIDs.length`